src/main/resources/i18n/messages.properties

```diff
@@ -514,7 +515,32 @@ UserDetailsPage.form.button.submit=Update User
 UserDetailsPage.createdDate=Created on {0}
 UserDetailsPage.modifiedDate=Modified on {0}
 UserDetailsPage.lastLogin=Last login on {0}
-UserDetailsPage.loading.message=Loading user details.
+
+UserSecurityPage.title=Security
+UserSecurityPage.alert.title=Missing Email Configuration
+UserSecurityPage.alert.description=Password reset is disabled, because IRIDA is not currently configured to send e-mails.
```

Comment:
Change this message to `Password reset is disabled due to this instance of IRIDA not having a configured mail server`

Reply:
Please see 453b0ef8bc524172456e276fddb553346c3c3cd8.